akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -101,6 +135,159 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
       .build().toByteArray
   }
 
+  private def serializeConfig(c: Config): Array[Byte] = {
+    c.root.render().getBytes(StandardCharsets.UTF_8)
```

Comment:
Perhaps use some `ConfigRenderOptions` to make it less verbose? Perhaps use the `concise` option?